frontendExternalSystem/src/main/java/com/virtuslab/gitmachete/frontend/externalsystem/project/MacheteProjectResolver.java

```diff
@@ -61,7 +61,8 @@ private Boolean hasSelectedGitMacheteTab(Project project) {
       val contentManager = toolWindow.getContentManagerIfCreated();
       if (contentManager != null) {
         val gitMacheteContent = contentManager.findContent("Git Machete");
-        return contentManager.getSelectedContent() == gitMacheteContent;
```

Comment:
Whoops okay so both sides `==` used to be `null` in a special case?

Reply:
yup